src/common/Navbar.js

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { useHistory } from 'react-router-dom';
+import NavbarItem from './NavbarItem';
+import TwweetBtn from './TweetBtn';
+import Logo from './Logo';
+import svg from '../style/images/logo.svg'
+import { connect } from 'react-redux';
+
+function Navbar(props) {
```

Comment:
You can do destruction to the props as part of params part so instead of Navbar(props) => NavBar({notificationsList, paramB, paramC...and more..})